### DISCUSSION TITLE
Tas Reforms :construction: :construction_site:

### DISCUSSION THREAD

**Discussion by ELPatitoLoco** [TAServers/issues#26] [Ideas]
Hello, with the last night events I wanted to get some things writen down here to keep track of them or to see if you agree.
So far last night we got this:
1. It would be good for the playerbase/ player count to get some general addons that unfortunatly people love,(ex: Outfitter, fishing mod, m9k) so we can appeal to a more general audience.
2. Get **ALL** ranks removed from isAdmin and IsSuperAdmin()
3. Remove Xander perms from uk server because of the trillions of complains about him and the  [reasons that were given yesterday](https://discord.com/channels/800234484649820163/800235133231300618/1184968169434140693)
4. Get the tas map done
5. Whait for the arcade project to be done
This isn't anything oficial, just wanted to make this and have somewhere to get tas users feedback on what I think could be improvments.

**Comment by d4v3cc** [comment]
AGREED

**Comment by Derpius** [comment]
We also need to fix the following issues:
- ~~Wiremod holoparent bypasses CPPI checks (this prevent’s most blinders, including xander’s)~~ Done
- ~~cleanupdiscon requires IsAdmin due to a missed admin check in the NADMOD.CDP function (cleanup disconnected players)~~ Done

> Remove Xander perms from uk server

If the wiremod issue is fixed and isadmin/issuperadmin is removed from *all* of us, I don’t think this is necessary anymore?

> get some general addons that unfortunatly people love

~~I’ll make a post in announcements today to get some addon suggestions (with the ground rules of no security issues, and no outfitter due to impact to players)~~ Done

**Comment by Derpius** [comment]
M9K (original) is added, and ULX groups (on the UK server for now) have been updated to be separate from `admin`/`superadmin`/`owner` in ULX, so no one (including me) has `IsAdmin` or `IsSuperAdmin` now

**Comment by ELPatitoLoco** [comment]
> > Remove Xander perms from uk server
> 
> If the wiremod issue is fixed and isadmin/issuperadmin is removed from *all* of us, I don’t think this is necessary anymore?
> 
Yeah, just wanted to get rid of the isAdmin() and isSuperAdmin()

**Comment by ELPatitoLoco** [comment]
> M9K (original) is added, and ULX groups (on the UK server for now) have been updated to be separate from `admin`/`superadmin`/`owner` in ULX, so no one (including me) has `IsAdmin` or `IsSuperAdmin` now

Then will we go back to the old admin system? Trusted admins get trusted and really trusted admins who have been there and helped get superadmin?

**Reply by Derpius** [reply]
No, the admin, superadmin and owner groups are default ULX groups that are used to determine `IsAdmin` etc. We're now using `trusted` and `system-operator` custom groups which inherit from `veteran`

**Reply by Derpius** [reply]
For context, previously `admin` was trusted and `owner` was system operator

**Reply by ELPatitoLoco** [reply]
Oh ok, thanks
